felix/config/config_params.go

```diff
@@ -182,6 +182,8 @@ type Config struct {
 	BPFDataIfacePattern                *regexp.Regexp    `config:"regexp;^((en|wl|ww|sl|ib)[Popsx].*|(eth|wlan|wwan).*|tunl0$|vxlan.calico$|wireguard.cali$|wg-v6.cali$)"`
 	BPFL3IfacePattern                  *regexp.Regexp    `config:"regexp;"`
 	BPFConnectTimeLoadBalancingEnabled bool              `config:"bool;true"`
```

Comment:
Does it make sense to leave this one without default? In fact, can we remove it here? (me not :100: sure)